acceptance/localcluster/localcluster.go

```diff
@@ -195,7 +213,7 @@ func (l *Cluster) initCluster() {
 	l.panicOnStop()
 
 	vols := map[string]struct{}{}
-	for i := range l.Nodes {
+	for i := 0; i < cap(l.Nodes); i++ {
```

Comment:
Iterating over `cap(l.Nodes)` seems delicate. I'd rather add a `numNodes` to the struct instead of hiding this in `cap(l.Nodes)` and rely on all interactions with this slice to preserve its backing array's capacity.